NGRValidator/NGRValidator/Extensions/NGRPropertyValidator+NGRImage.h

```diff
@@ -0,0 +1,78 @@
+//
+//  NGRPropertyValidator+NGRImage.h
+//  NGRValidator
+//
+//  Created by Krzysztof Kapitan on 23.03.2017.
+//
+//
+
+#import "NGRPropertyValidator.h"
+#import "NGRImage.h"
+
+@interface NGRPropertyValidator (NGRImage)
```

Comment:
Hmm I was wondering whether this is good or bad idea. Maybe it would be better to add such validation on NSData since there is no explicit way to get byte size from UIImage/NSImage apart from making JPG/PNG representation out of it? So the client could decide which data representation suits his need and then validate against its size